src/fmu/ensemble/realizationcombination.py

```diff
@@ -117,16 +120,36 @@ def get_df(self, localpath):
             return result.reset_index()
         return result.dropna().to_dict()
 
-    def to_virtual(self):
+    def to_virtual(self, keyfilter=None):
         """Evaluate the current linear combination and return as
         a virtualrealizatione.
```

Comment:
Technically not edited in this commit, but enclosed by modifications:
```suggestion
        a VirtualRealization.
```